labelbox/schema/project.py

```diff
@@ -688,35 +690,40 @@ def create_batch(self,
             raise ValueError("Project must be in batch mode")
 
         dr_ids = []
-        for dr in data_rows:
-            if isinstance(dr, Entity.DataRow):
-                dr_ids.append(dr.uid)
-            elif isinstance(dr, str):
-                dr_ids.append(dr)
-            else:
-                raise ValueError("You can DataRow ids or DataRow objects")
+        if data_rows is not None:
+            for dr in data_rows:
+                if isinstance(dr, Entity.DataRow):
+                    dr_ids.append(dr.uid)
+                elif isinstance(dr, str):
+                    dr_ids.append(dr)
+                else:
+                    raise ValueError("You can DataRow ids or DataRow objects")
```

Comment:
I know it is unrelated to this PR, but should pretty easy to improve. The error text is weird and would be cool to add "provide" word in there =)

Reply:
Good catch, updated!